sklearn/tree/tests/test_tree.py

```diff
@@ -1693,19 +1693,104 @@ def test_no_sparse_y_support(name):
 
 
 def test_mae():
-    # check MAE criterion produces correct results
-    # on small toy dataset
+    ''' 
```

Comment:
replace `'''` by `"""` and put the title on the first line.

The example is verbose but at least it explains the rational behind the number.